### PR TITLE
Api answers

### PR DESCRIPTION
**Pull request by patrikmolsson**
Connected api to list activity.

Note: The new server code is needed to use these fixes.

### CODE REVIEW

File: app/src/main/java/com/pinomg/determinator/api/ApiHandler.java [hunk 1]
```diff
@@ -23,7 +23,7 @@
  */
 public class ApiHandler {
     
-    private static final String BASE_URL = "http://95.80.41.105:8004/determinator_server/";
+    private static final String BASE_URL = "http://192.168.0.11/pinomg/";
```
Comment (ollelindeman) [inline app/src/main/java/com/pinomg/determinator/api/ApiHandler.java:26]: Ey!

File: app/src/main/java/com/pinomg/determinator/CustomAdapter.java [hunk 2]
```diff
@@ -31,7 +32,8 @@
     public CustomAdapter(Context context, List<Poll> polls) {
         mInflater = (LayoutInflater) context.getSystemService(Context.LAYOUT_INFLATER_SERVICE);
 
-        items = new HashMap<Integer, List<Poll>>();
```
Comment (ollelindeman) [inline app/src/main/java/com/pinomg/determinator/CustomAdapter.java:34]: Behövs detta inte alltså?

File: app/src/main/java/com/pinomg/determinator/api/ApiHandler.java (hunk 1 above)
Comment (patrikmolsson) [inline app/src/main/java/com/pinomg/determinator/api/ApiHandler.java:26]: Lol. Did it on purpose to show that the new server code is needed.

File: app/src/main/java/com/pinomg/determinator/CustomAdapter.java (hunk 2 above)
Comment (patrikmolsson) [inline app/src/main/java/com/pinomg/determinator/CustomAdapter.java:34]: Redundant kod.

Comment (patrikmolsson) [inline app/src/main/java/com/pinomg/determinator/CustomAdapter.java:34]: Ersatt med items = new HashMap<>();

File: app/src/main/java/com/pinomg/determinator/api/ApiHandler.java (hunk 1 above)
Comment (ollelindeman) [inline app/src/main/java/com/pinomg/determinator/api/ApiHandler.java:26]: Okey, but then we have to change it back after the server is updated. And that's annoying! ;)

Comment (patrikmolsson) [inline app/src/main/java/com/pinomg/determinator/api/ApiHandler.java:26]: D:

File: app/src/main/java/com/pinomg/determinator/CustomAdapter.java (hunk 2 above)
Comment (ollelindeman) [inline app/src/main/java/com/pinomg/determinator/CustomAdapter.java:34]: coolt!

**Pull request merged**